shared/src/main/java/com/faforever/neroxis/biomes/Biomes.java

```diff
@@ -78,7 +75,7 @@ public static Biome loadBiome(String folderPath) {
                     String.format("An error occurred while loading %s LightingSettings\n", folderPath), e);
         }
 
-        return new Biome(terrainMaterials.getName(), terrainMaterials, propMaterials, decalMaterials, waterSettings,
+        return new Biome(BiomeName.getByValue(terrainMaterials.getName()), terrainMaterials, propMaterials, decalMaterials, waterSettings,
```

Comment:
You can just use the passed in enum if changed as suggested

Reply:
the terrainmaterials get compiled from JSON. Does using enums work there without any further changes?

Reply:
Does it even make sense that the terrainmaterials specify the name again? It seems redundant

Reply:
Ah I see, I just take the parameter from the very top of the function